app/config/views.yml

```diff
@@ -1,6 +1,7 @@
 ezpublish:
     system:
         default:
+            pagelayout: "pagelayout.html.twig"
             user:
                 layout: pagelayout.html.twig
```

Comment:
It could be made irrelevant, yes, but more often then not, login pages have a different layout, especially when accessing a backend of some sort, so this could still be useful.